pkg/ui/ccl/src/views/clusterviz/containers/map/nodeCanvas.tsx

```diff
@@ -71,15 +73,15 @@ export class NodeCanvas extends React.Component<NodeCanvasProps, NodeCanvasState
     window.removeEventListener("resize", this.debouncedOnResize);
   }
 
-  renderContent() {
+  renderContent(asMap: boolean) {
```

Comment:
well that's an unpleasant change.  boolean parameters are bad.  perhaps just move the conditional up to the parent?

Reply:
Haven't seen an elegant way to get rid of this. Let's look at it together.